src/main/java/com/revature/lostchapterbackend/service/CheckoutService.java

```diff
@@ -20,11 +20,11 @@
 @Service
 public class CheckoutService {
 
-	double totalPriceOfEachBook = 0.0;
-	double totalPrice = 0.0;
-	double subTotal = 0.0;
-	int booksRemaining = 0;
-	final double TAX = 0.06; // 6%
+	static double totalPriceOfEachBook = 0.0;
+	static double totalPrice = 0.0;
+	static double subTotal = 0.0;
+	static int booksRemaining = 0;
+	final static double tax = 0.06; // 6%
 
```

Comment:
Having these static fields as part of the business logic could result in thread safety issues since if you have multiple users trying to check out at the same time, they're going to be sharing the fields above.

Realistically you'd have to do something like use ThreadLocal (similar to what I did in the TestNG w/ Selenium demo) if you'd like to approach it this way, or have the variables as local method scoped variables.

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/ThreadLocal.html